lib/bolt/transport/orch.rb

```diff
@@ -124,9 +131,13 @@ def run_task_job(targets, task, arguments)
         end
 
         begin
-          results = create_client.run_task(body)
+          results = create_client(targets.first.options).run_task(body)
```

Comment:
This might eventually benefit from some connection pooling, similar to the other transports. Seems fine to leave that until later.

Reply:
That will require work in the client itself as well